org.viking.Viking.yml

```diff
@@ -0,0 +1,183 @@
+app-id: org.viking.Viking
+# https://docs.flatpak.org/en/latest/available-runtimes.html
+# NB Viking doesn't use any Gnome specifics so freedesktop should suffice for the build
+# Albeit Yelp for help is Gnome only
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+command: viking
+rename-icon: viking
+rename-desktop-file: viking.desktop
+finish-args:
+  - --share=network
+  - --system-talk-name=org.freedesktop.GeoClue2
+  - --share=ipc
+  - --socket=x11
+# Allow to read/save anywhere (subject to normal permissions)
+  - --filesystem=host
```

Comment:
How common is it to save/load files from outside of /home? It's not much but if it's not very popular, `home` would be slightly better here.

Reply:
Very common.
One of the primary uses of Viking is to read and write GPX files from GPS devices.
Many of the popular GPS devices (e.g. Garmin Etrex) use a USB filesystem for data transfer of Tracks, Routes and Waypoints.
Read is mostly used for accessing recordings of tracks of where one has been (and Waypoints created on the GPS device); and write is for putting a planned route/track on to the device that you subsequently may follow when one is out and about (and maybe Waypoints of Points Of Interest (POIs)to visit - e.g. GeoCaches).
Under Linux the GPS Device is mounted just like any external USB drive and this is often located in /media/<user>/<DeviceName>/ by default.
So IFAIK this sits outside any $XDG_* scheme, so I'm afraid $HOME would be too limiting.

PS Thanks for the review.